client/components/dashboard.js

```diff
@@ -1,45 +1,97 @@
 import React, {Component} from "react";
+import Home from './home';
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import {Link} from 'react-router';
+import GroupActions from '../actions/group';
+import SideNav from './sidenav';
+import ReactPaginate from 'react-paginate';
 
+const LIMIT = 6;
+class Dashboard extends Component {
+  setGroupIDHandler(groupID, groupName) {
+    localStorage.setItem('groupID', groupID);
+    localStorage.setItem('groupName', groupName);
+  }
+  componentDidMount() {
+    this.props.getTotalGroups(JSON.parse(localStorage.user).user)
+    if (localStorage.groupOffset) {
+      this.props.getGroups(JSON.parse(localStorage.user).user, localStorage.groupOffset, LIMIT);
+    } else {
+      this.props.getGroups(JSON.parse(localStorage.user).user, 0, LIMIT);
+    }
+  }
+
+  handlePageClick(data) {
+    let selected = data.selected;
+    let offset = Math.ceil(selected * LIMIT);
+    localStorage.setItem('groupOffset', offset);
```

Comment:
Do you really need to keep this in the localstorage? What if a user tampers with it since it on the client side?